### ISSUE TITLE
Print: Entry, ":CFBundleIdentifier", Does Not Exist

### ISSUE THREAD

**Issue by abuammar** [futurice/pepperoni-app-kit#147]
The following build commands failed:
	CompileC /Users/ws1/Code/mobile/ios/build/Build/Intermediates/AirMaps.build/Debug-iphonesimulator/AirMaps.build/Objects-normal/x86_64/AIRMapPolyline.o AirMaps/AIRMapPolyline.m normal x86_64 objective-c com.apple.compilers.llvm.clang.1_0.compiler
	CompileC /Users/ws1/Code/mobile/ios/build/Build/Intermediates/AirMaps.build/Debug-iphonesimulator/AirMaps.build/Objects-normal/x86_64/AIRMapUrlTileManager.o AirMaps/AIRMapUrlTileManager.m normal x86_64 objective-c com.apple.compilers.llvm.clang.1_0.compiler
(2 failures)
Installing build/Build/Products/Debug-iphonesimulator/WS1.com.app
An error was encountered processing the command (domain=NSPOSIXErrorDomain, code=2):
Failed to install the requested application
An application bundle was not found at the provided path.
Provide a valid path to the desired application bundle.
Print: Entry, ":CFBundleIdentifier", Does Not Exist

Command failed: /usr/libexec/PlistBuddy -c Print:CFBundleIdentifier build/Build/Products/Debug-iphonesimulator/WS1.com.app/Info.plist
Print: Entry, ":CFBundleIdentifier", Does Not Exist

**Comment by tino-junge** [comment]
Hi @abuammar. It looks like you are trying to run the iOS build without installing pods first. So to solve your issue try to run

```
(cd ios; pod install)
react-native run-ios
```

from the repository root directory.

**Comment by abuammar** [comment]
i solved the issue by the following below steps:

Open your project in Xcode, right click on Libraries and click Add Files to "Your Project Name" Look under node_modules/react-native-maps/ios and add AIRMaps.xcodeproj.
Add libAIRMaps.a to `Build Phases -> Link Binary With Libraries.
Click on AIRMaps.xcodeproj in Libraries and go the Build Settings tab. Double click the text to the right of Header Search Paths and verify that it has $(SRCROOT)/../../react-native/React as well as $(SRCROOT)/../../react-native/Libraries/Image - if they aren't, then add them. This is so Xcode is able to find the headers that the AIRMaps source files are referring to by pointing to the header files installed within the react-native node_modules directory.
Whenever you want to use it within React code now you can: var MapView = require('react-native-maps');

**Comment by jiawang1** [comment]
hi @tino-junge :  since I am brand new for mobile development. I do not know why I have to run 
````code
pod install
````
since for vanilla react-native, there is no this step. I guess pepperoni depends on some native components. could you let me know what are they? Maybe they are not necessary for my project.

thanks.

**Comment by tino-junge** [comment]
hi @jiawang1, you're right it's not really necessary for pepperoni anymore since we removed Auth0 as a native dependency. `pod install` refers to Cocoapods which are used to maintain native iOS libraries and frameworks. We added it because we wanted to add Auth0 to pepperoni to integrate a solution for user management and authentication. However, after the launch of pepperoni we've learned that most users don't need it and it makes the whole setup just more complicated. We still kept the cocoapods dependency which allowed users to still add own native extensions. Right now I feel it's not really necessary either so we might just remove it soon as well to smoothen the setup process for pepperoni even more :)
